frontend/src/metabase/entities/questions.js

```diff
@@ -100,7 +100,14 @@ const Questions = createEntity({
   },
 
   objectSelectors: {
-    getName: question => question && question.name,
+    getName: question => {
+      // When a question is from `getObject`, it will be a `metabase-lib/Question` object wrapped in EntityWrapper.
+      if (question?._card) {
```

Comment:
Not sure if checking `_card` directly would make sense here.

Reply:
Maybe `question instanceof Question`?

Reply:
Fixed in https://github.com/metabase/metabase/pull/33432/commits/fc97f958d1dc46612beab96202135ee7b0327745